geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl

```diff
@@ -13,6 +13,17 @@ Information
    a `.env.mako` file where we have the environment variables used in the Docker compose files,
    `COMPOSE_PROJECT_NAME` contains the default name of the project.
 
+Changes to apply
+----------------
+
+1. The ngeo modules import have change, the major chang that will consern you is that we dirrectly
```

Comment:
1. The ngeo modules import has changed. The major change that will concern you is that we drrectly
export the module, therefore, in the angular module declaration, you should replace `<module>.module.name`
   by `<module>.name`.